.github/workflows/ai-lab-e2e-nightly-windows.yaml

```diff
@@ -48,25 +48,35 @@ on:
         type: string
         required: true
       podman_remote_url:
-        default: 'https://github.com/containers/podman/releases/download/v5.0.3/podman-remote-release-windows_amd64.zip'
+        default: 'https://github.com/containers/podman/releases/download/v5.2.0/podman-remote-release-windows_amd64.zip'
```

Comment:
Worth trying replacing v5.2.0 by latest for better maintainibilty

Reply:
I would say this workflow focuses on nightly version of the extension and tests it against latest releases of podman-desktop and podman. This values of the input is picked when run manually, so it is not that important here after all.